### DISCUSSION TITLE
Targets' source does not accept globs?

### DISCUSSION THREAD

**Discussion by jankocian** [parcel-bundler/parcel#8329] [Q&A]
Hi all,
in my package.json, I define multiple entrypoints as targets` sources. Unfortunately, it seems like I cannot use an asterisk to select all the files in a certain folder.

`"source": ["./src/assets/legacy-styles/pages/*"],` gives me an error `@parcel/core: src/assets/legacy-styles/pages/* does not exist.`.

I really want to get rid of the...
```
"targets": {
    "legacy-styles-pages": {
      "source": ["./src/assets/legacy-styles/pages/airtable.less", "./src/assets/legacy-styles/pages/career.less", "./src/assets/legacy-styles/pages/contact.less", "./src/assets/legacy-styles/pages/features.less", "./src/assets/legacy-styles/pages/changelog.less", "./src/assets/legacy-styles/pages/pricing.less", "./src/assets/legacy-styles/pages/pro-partnery.less", "./src/assets/legacy-styles/pages/roadmap.less", "./src/assets/legacy-styles/pages/webinar.less"],
      "distDir": "./dist/legacy-styles/"
    }
  },
```
Any ideas? :)

Thanks a lot,
Jan

**Comment by mischnic** [comment]
See https://github.com/parcel-bundler/parcel/issues/3821
